string EntryName

#these are array fields the first
#array that is not zero length defines the datatype 
#of the menu entry
#the first value of it the value of the entry
#any other value or array comming later is IGNORED!

float64[] double
float32[] float
bool[]    boolean
int8[]    integer8
uint8[]   uinteger8
int16[]   integer16
uint16[]  uinteger16
int32[]   integer32
uint32[]  uinteger32
int64[]   integer64
uint64[]  uinteger64
